<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PolinomMainClass</class>
 <widget class="QMainWindow" name="PolinomMainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PolinomMain</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lbOperation">
          <property name="text">
           <string>Operation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbOperation">
          <item>
           <property name="text">
            <string>Polinom + Polinom</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Polinoms - Polinom</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Polinom * constant</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Polinom + constant</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbOperand1">
          <property name="text">
           <string>Polinom1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edOp1">
          <property name="text">
           <string>-2XYZ-3XY+8X^2Z^3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbOperand2">
          <property name="text">
           <string>Polinom2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edOp2">
          <property name="text">
           <string>XYZ+3XY+9X^2Z</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="teResult"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pbRun">
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionActionText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ActionText</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PolinomMain.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>slotRun()</slot>
  <slot>slot2()</slot>
 </slots>
</ui>
